Makefile

```diff
@@ -189,8 +189,19 @@ payload: bots
 	bots/image-download $(PAYLOAD)
 
 .PHONY: images
-images: bots
-	bots/image-download $(TEST_OS) debian-stable ubuntu-stable fedora-41
+images: bots cloud-images iso-images
+
+.PHONY: cloud-images
+	bots/image-download debian-stable ubuntu-stable fedora-41
+
+.PHONY: iso-images
+	# If a compose if specified download from the compose otherwise
+	# download the ISO from Cockpit image server
+	if [ -n "$TEST_COMPOSE" ]; then \
+		test/download-iso $TEST_OS $TEST_COMPOSE; \
+	else \
+		bots/image-download $TEST_OS; \
```

Comment:
The dot will break the command.

Reply:
test/download-iso "$(TEST_OS)" "$(TEST_COMPOSE)"; \    

This should work, right?

Reply:
Err, which dot... I'm not crazy I  did see it. But oops, I just scratched off a speck of dirt from my monitor which happened to be on a very strategic place :see_no_evil: Sorry!